src/pages/Profile/Profile.css

```diff
@@ -2,17 +2,45 @@
     height: 100vh;
     width: 100vw;
     display: flex;
+    flex-wrap: wrap;
     align-items: center;
+    justify-content: center;
+    font-size: 2.25rem;
 }
 
-/* TODO: Add your styles here
-         for the Profile page.
-         Hint: The lines above ensure that
-         the page takes up the full width
-         and height of the browser window.
-         This will help you when you get started.
-    */
+.main-profile {
+    max-width: 90vw;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 3.5rem;
+}
 
+.avatar {
+    width: 27rem;
+    height: 27rem;
+    border-radius: calc(27rem / 2);
+    background: #BFBFBF;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-shrink: 0;
+}
 
+.avatar img {
```

Comment:
generally it's not very good practice to directly select tags either outside of global styles, if we need to style images we should apply a unique `className` to them based on how far we want to generalize the styles